### PR TITLE
fix styles for mobile

### PR DESCRIPTION
**Pull request by ivanheral**

### CODE REVIEW

**Comment by Havunen** [comment]
Shouldn't we write styles mobile first

File: src/components/common/Header.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ export default class Header extends Component {
         }
         return (
         <div className="menu">
-            <div className="container row">
+            <div className="container">
+            <div className="row">
```
Comment (yarhtut) [inline src/components/common/Header.js]: Indentation

File: src/components/common/Header.js [hunk 2]
```diff
@@ -49,7 +50,7 @@ export default class Header extends Component {
                     <a target="_blank" rel="noopener" href="https://github.com/trueadm/inferno/issues">Contribute</a>
                     <a target="_blank" rel="noopener" href="https://github.com/infernojs">Github</a>
                 </nav>
-            </div>
+            </div></div>
```
Comment (yarhtut) [inline src/components/common/Header.js]: Indentation

File: src/assets/css/modules/_repl.scss [hunk 3]
```diff
@@ -18,4 +18,9 @@
         display: block;
         margin: 1rem auto;
     }
+    .CodeMirror pre {
+         @include media-query($on-small) {
+        font-size: 13px;
```
Comment (yarhtut) [inline src/assets/css/modules/_repl.scss]: Indentation

**Pull request merged**